docs/sdk/guides/03-build-your-first-cross-chain-dapp.md

```diff
@@ -2,165 +2,129 @@
 sidebar_position: 3
 ---
 
-# Build your first cross-chain Dapp
+# Build Your First Cross-chain Dapp
 
-We will use [hardhat](https://hardhat.org/) to build our first cross-chain Dapp. The Dapp will be deployed to the Pangoro Smart Chain, then call the `remark_with_event` on the [Pangolin](https://docs.crab.network/evm-compatible-crab-smart-chain/get-started/darwinia-pangolin) remotely.
+This Dapp allows you to call Pangolin's `remark_with_event` cross-chain from Pangoro.
```

Comment:
Please add more descriptions of Pangolin and Pangoro? For these docs readers, I guess they don't know what Pangolin is.

Reply:
what are Pangolin, Pangoro, remark_with_event dispatch call

Reply:
Execute remote smart contract call
Dispatch remote substrate call

Reply:
standlone example repo

Reply:
runnable example